### PR TITLE
CHORE: Align logging in EN/Pokémon Scripts after PRs

### PR DESCRIPTION
**Pull request by XK9274**
Minor fix to logging

### CODE REVIEW

File: static/build/.tmp_update/script/easynetplay/netplay_client.sh [hunk 1]
```diff
@@ -260,22 +261,14 @@ build_infoPanel_and_log() {
 	local title="$1"
 	local message="$2"
 
-	if [ $LOGGING -eq 1 ]; then
-		echo "$(date) GLO::Easy_Netplay: Stage: $title Message: $message" >> $sysdir/logs/easy_netplay.log
-	fi
+	log
```
Comment (Aemiii91) [inline static/build/.tmp_update/script/easynetplay/netplay_client.sh:264]: log... nothing?

File: static/build/.tmp_update/script/easynetplay/netplay_server.sh [hunk 2]
```diff
@@ -182,22 +182,14 @@ build_infoPanel_and_log() {
 	local title="$1"
 	local message="$2"
 
-	if [ $LOGGING -eq 1 ]; then
-		echo "$(date) GLO::Easy_Netplay: Stage: $title Message: $message" >> $sysdir/logs/easy_netplay.log
-	fi
+	log
```
Comment (Aemiii91) [inline static/build/.tmp_update/script/easynetplay/netplay_server.sh:185]: log... nothing?

File: static/build/.tmp_update/script/easynetplay/pokemon_client.sh [hunk 3]
```diff
@@ -533,22 +533,14 @@ build_infoPanel_and_log() {
 	local title="$1"
 	local message="$2"
 
-	if [ $LOGGING -eq 1 ]; then
-		echo "$(date) GLO::Pokemon_Netplay: Stage: $title Message: $message" >> $sysdir/logs/easy_netplay.log
-	fi
+	log
```
Comment (Aemiii91) [inline static/build/.tmp_update/script/easynetplay/pokemon_client.sh:536]: log... nothing?

File: static/build/.tmp_update/script/easynetplay/pokemon_server.sh [hunk 4]
```diff
@@ -633,22 +633,14 @@ build_infoPanel_and_log() {
 	local title="$1"
 	local message="$2"
 
-	if [ $LOGGING -eq 1 ]; then
-		echo "$(date) GLO::Pokemon_Netplay: Stage: $title Message: $message" >> $sysdir/logs/easy_netplay.log
-	fi
+	log
```
Comment (Aemiii91) [inline static/build/.tmp_update/script/easynetplay/pokemon_server.sh:636]: log... nothing?

Comment (XK9274) [reply static/build/.tmp_update/script/easynetplay/pokemon_server.sh]: https://github.com/OnionUI/Onion/pull/1047/commits/64c08e8dede8cd98a6b3682a75900d20d30d6b06

File: static/build/.tmp_update/script/easynetplay/pokemon_client.sh (hunk 3 above)
Comment (XK9274) [reply static/build/.tmp_update/script/easynetplay/pokemon_client.sh]: https://github.com/OnionUI/Onion/pull/1047/commits/64c08e8dede8cd98a6b3682a75900d20d30d6b06

File: static/build/.tmp_update/script/easynetplay/netplay_server.sh (hunk 2 above)
Comment (XK9274) [reply static/build/.tmp_update/script/easynetplay/netplay_server.sh]: https://github.com/OnionUI/Onion/pull/1047/commits/64c08e8dede8cd98a6b3682a75900d20d30d6b06

File: static/build/.tmp_update/script/easynetplay/netplay_client.sh (hunk 1 above)
Comment (XK9274) [reply static/build/.tmp_update/script/easynetplay/netplay_client.sh]: https://github.com/OnionUI/Onion/pull/1047/commits/64c08e8dede8cd98a6b3682a75900d20d30d6b06

**Review by Aemiii91 — APPROVED** [review]

**Pull request merged**